src/ontology/pathgo-edit.owl

```diff
@@ -1771,6 +1775,7 @@ build out subclasses</skos:editorialNote>
     <owl:Class rdf:about="http://purl.obolibrary.org/obo/pathgo/PATHGO_0000204">
         <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/pathgo/PATHGO_0000199"/>
         <rdfs:label>phagolysosome fusion disruption factor</rdfs:label>
+        <skos:definition>A gene product that modulates host cell endomembrane dynamics by dirupting phagosome fusion.</skos:definition>
```

Comment:
fix spelling error- disrupting
change phagosome to phagolysosome

Reply:
fixed!